README.md

```diff
@@ -1,49 +1,53 @@
 [![Build Status](https://travis-ci.org/RedHatInsights/insights-frontend-starter-app.svg?branch=master)](https://travis-ci.org/RedHatInsights/insights-frontend-starter-app)
 
+> NOTE: This project is still under active development
+
 # Application Services UI
 
-mk-ui is based on the https://github.com/RedHatInsights/insights-frontend-starter-app.git which is React.js starter app for Red Hat Insights products that includes Patternfly 4 and shared cloud.redhat.com services
+Application Services UI is based on the https://github.com/RedHatInsights/insights-frontend-starter-app.git which is React.js starter app for Red Hat Insights products that includes Patternfly 4 and shared cloud.redhat.com services
 
 ## Services 
 
 - Managed Kafka UI (https://github.com/bf2fc6cc711aee1a0c2a/kas-ui)
 - Kafka instance UI (https://github.com/bf2fc6cc711aee1a0c2a/kafka-ui)
-- Guides (https://github.com/bf2fc6cc711aee1a0c2a/guides)
 - Service Registry UI (https://github.com/bf2fc6cc711aee1a0c2a/srs-ui)
+- Guides (https://github.com/redhat-developer/app-services-guides)
 
 ## Running Project
 
-### Prerequisites
+Project can be run in two modes:
 
-- Docker 
-- NVM (node version manager)
+- Using compiled federated components (preview)
+- Running federated modules local (dev)
 
-## Initial dev setup
+## Running project with compiled components
 
+Run:
 ```
-./scripts/checkout-repos.sh
+npm install
+npm run start:dev
 ```
-This will download all required repositories for development and setup insights container
 
-## Running project
+Go to https://prod.foo.redhat.com:1337/beta/application-services/streams/kafkas
+
+## Running project with compiled components
+
+#### Setting up all repositories for development
+
+First we need to download all external repositories by running script.
 
 ```
-./scripts/run.sh
+./scripts/checkout-repos.sh
 ```
 
-Go to https://prod.foo.redhat.com:1337/beta/application-services/streams/kafkas
+After that we can run any of the federated UI component in `./modules` folder and it will be automatiocally discovered by application services-ui
```

Comment:
```suggestion
After that we can run any of the federated UI component in `./modules` folder and it will automatically be used by the app-services-ui
```